src/solvers/dgmulti/sbp.jl

```diff
@@ -257,4 +257,83 @@ function StartUpDG.inverse_trace_constant(rd::RefElemData{NDIMS, ElementType, Ap
   #     Chan, Jesse, et al (2016), https://doi.org/10.1016/j.jcp.2016.04.003
   # for more details (specifically, Appendix A.1, Theorem A.4).
   return NDIMS * max_eigenvalue
-end
\ No newline at end of file
+end
+
+# type alias for specializing on a periodic SBP operator
+const DGMultiPeriodicFDSBP{ApproxType, ElemType} =
+  DGMulti{NDIMS, ElemType, ApproxType, SurfaceIntegral, VolumeIntegral} where {NDIMS, ElemType, ApproxType<:SummationByPartsOperators.AbstractPeriodicDerivativeOperator, SurfaceIntegral, VolumeIntegral}
+
+"""
+    CartesianMesh(dg::DGMulti)
+
+Constructs a single-element `mesh::AbstractMeshData` for a single periodic element given
+a DGMulti with `approximation_type` set to a periodic (finite difference) SBP oerator from 
+SummationByPartsOperators.jl.
```

Comment:
```suggestion
Constructs a single-element [`VertexMappedMesh`](@ref) for a single periodic element given
a DGMulti with `approximation_type` set to a periodic (finite difference) SBP operator from 
SummationByPartsOperators.jl.
```
Or do you plan to use another type for the returned mesh?